### PR TITLE
Add requisite libraries for ArrayFire

### PR DESCRIPTION
**Pull request by ranjanan**
ArrayFire fails on PackageEvaluator. This should make it work.

### CODE REVIEW

**Comment by tkelman** [comment]
can't the package be made to handle installation automatically?

**Comment by ranjanan** [comment]
You need a login and a password to download the binary. I'm pulling from their Jenkins CI with persmission.

File: scripts/setup.sh [hunk 1]
```diff
@@ -89,6 +89,13 @@ export PYTHON=""
 sudo apt-get install wamerican
 # Need xmllint (and others?) for XMLDict.jl
 sudo apt-get install libxml2-utils
+# ArrayFire
+# ArrayFire
```
Comment (tkelman) [inline scripts/setup.sh]: don't duplicate this

**Comment by tkelman** [comment]
ah, ok. why are they using atlas?

**Comment by ranjanan** [comment]
It is listed as a pre-requisite in their docs. I think this is because it doesn't ship with a BLAS, so it tries to link to system BLAS by default (and that step is just in case a system BLAS isn't installed). You can build by linking to OpenBLAS, but I haven't tried linking the binary with `libopenblas.so`

**Comment by tkelman** [comment]
Can you test if it works with openblas instead? Would rather not install things we don't need. And why do you need a login to download a binary, isn't arrayfire open source now?

**Comment by ranjanan** [comment]
Yes it is, in that you can clone the library and build it from source. That's going to take time though. And the reason I don't automate building from source is that there are many ways you would do that depending on the hardware you have.

**Comment by ranjanan** [comment]
For a minimum install, it turns out you don't need any of those libraries, so I'm removing ATLAS, FreeImage, etc.

**Comment by ranjanan** [comment]
Fails because of [this issue](https://github.com/JuliaLang/julia/issues/17484)

**Comment by tkelman** [comment]
ArrayFire is really more than 650 MB? Why is it such a large download?

**Comment by ranjanan** [comment]
They're bundling a number of libraries with it. Like CLBLAS, CLFFT, etc.

**Comment by tkelman** [comment]
We won't be testing any of those on PkgEval, will we? They don't have a smaller CPU-only version?

**Comment by ranjanan** [comment]
No, they don't have anything lighter than this.

**Comment by tkelman** [comment]
I guess we can try this, but adding 650+ MB (that's just the download, then duplicated or more once installed?) to each worker VM might be a bit much. It's an open-source library, I think the best path forward is going to be just to build a CPU-only binary ourselves for testing purposes.

**Comment by tkelman** [comment]
Do you know if the shell script deletes itself after it's done?

**Comment by ranjanan** [comment]
No it doesn't. Should I throw in a command to delete it after extraction?

**Comment by tkelman** [comment]
Yeah, I think that would be good.

**Pull request merged**